styles.css

```diff
@@ -0,0 +1,25 @@
+body {
```

Comment:
It's better not to give properties for body tags, instead add all body content except for header and footer in main tag.